src/backend/rest.ts

```diff
@@ -199,6 +199,11 @@ export default class RestApiBackend implements Backend {
         'The data source specified the Checkmk Raw Edition, but a Checkmk commercial edition was detected. Some functionality may not be available. Choose commercial editions in the data source settings to enable all features.'
       );
     }
+    if (result.data.edition === 'cse') {
+      throw new Error(
+        'The data source specifed is Checkmk Cloud (SaaS). The grafana plugin is currently not supported for that edition.'
```

Comment:
The user can not specify "Checkmk Cloud (SaaS)" as the edition. i would suggest the following error message:
"Detected Checkmk Cloud (SaaS). Can not query data from Checkmk Cloud (SaaS)."